web/src/condense.ts

```diff
@@ -94,19 +113,19 @@ export function collapse($row) {
 
     message_flags.save_collapsed(message);
 
-    const process_row = function process_row($row) {
+    const process_row = function process_row($row: JQuery) {
         $row.find(".message_content").addClass("collapsed");
         show_more_link($row);
     };
 
     // We also need to collapse this message in the home view
-    const $home_row = message_lists.home.get_row(rows.id($row));
+    const $home_row = message_lists.home.get_row(id);
 
     process_row($row);
     process_row($home_row);
 }
 
-export function toggle_collapse(message) {
+export function toggle_collapse(message: Message) {
```

Comment:
This function definition is lacking a return type (`void`).This will throw ESlint error.